plugins/node/instrumentation-mongoose/test/mongoose.test.ts

```diff
@@ -63,7 +65,13 @@ describe('mongoose instrumentation', () => {
     await mongoose.connection.close();
   });
 
-  beforeEach(async () => {
+  beforeEach(async function mongooseBeforeEach() {
+    // Skipping all tests in beforeEach() is a workaround. Mocha does not work
```

Comment:
On pg tests there are some nested describes, and there there is only skip on the `before` and not on the before each, but is a little different:

```before(async function () {
    const skip = () => {
      // this.skip() workaround
      // https://github.com/mochajs/mocha/issues/2683#issuecomment-375629901
      this.test!.parent!.pending = true;
      this.skip();
    };

    if (!shouldTest) {
      skip();
    }
 ```
 
 maybe adding those extra lines help in your case too?

Reply:
Thanks, I'll take a look.

This hunk that I added here was copying the same pattern from some of the instr-mongodb tests:

```
plugins/node/opentelemetry-instrumentation-mongodb/test/mongodb-v5-v6.test.ts
86:    // Skipping all tests in beforeEach() is a workaround. Mocha does not work

plugins/node/opentelemetry-instrumentation-mongodb/test/mongodb-v4.test.ts
77:    // Skipping all tests in beforeEach() is a workaround. Mocha does not work

plugins/node/opentelemetry-instrumentation-mongodb/test/mongodb-v3.test.ts
76:    // Skipping all tests in beforeEach() is a workaround. Mocha does not work

plugins/node/opentelemetry-instrumentation-mongodb/test/mongodb-v4-v5-v6.metrics.test.ts
106:    // Skipping all tests in beforeEach() is a workaround. Mocha does not work
```